packages/ckeditor5-engine/src/view/placeholder.ts

```diff
@@ -37,10 +39,11 @@ const documentPlaceholders = new WeakMap<Document, Map<Element, PlaceholderConfi
  * editable root elements.
  * @param options.keepOnFocus If set `true`, the placeholder stay visible when the host element is focused.
  */
-export function enablePlaceholder( { view, element, isDirectHost = true, keepOnFocus = false }: {
+export function enablePlaceholder( { view, element, text, isDirectHost = true, keepOnFocus = false }: {
```

Comment:
You missed documenting it in API docs. If it's in the API, it should be listed as a param - however, with clear information that it is deprecated.